index.html

```diff
@@ -214,5 +214,55 @@ <h1>About Me</h1>
         </div>
       </section>
     </main>
+    <footer>
+      <section class="get-in-touch">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </section>
+      <form action="https://formspree.io/f/meqdgwge" method="POST">
+        <div class="form-group">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input type="email" name="email" placeholder="Email" required />
+          <textarea
+            cols="33"
+            rows="3"
+            name="message"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+        </div>
+        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Please as required by the previous reviewer kindly add button interaction like (hover, active when it's pressed, disabled, and enable "default state") but it is not the case for you all your buttons don't have interaction when I click or when I put the cursor to the buttons nothing happens. Please kindly fix that 🙃 🙃 

<img width="693" alt="Screen Shot 2022-09-28 at 11 47 41" src="https://user-images.githubusercontent.com/74501523/192747865-680c2349-9275-445a-9de2-6621b58acbe7.png">